src/__stories__/title-story.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import {Title1, Title2, Stack, TextLink} from '..';
+
+export default {
+    title: 'Components/Others/Title',
+};
+
+type Args = {title: string; linkText: string};
+
+export const Default: StoryComponent<Args> = ({title, linkText}) => (
+    <Stack space={32}>
+        <Title1
+            as="h3"
+            right={linkText ? <TextLink href="https://google.com">{linkText}</TextLink> : undefined}
+        >
+            {title}
+        </Title1>
+        <Title2
+            as="h2"
+            right={linkText ? <TextLink href="https://google.com">{linkText}</TextLink> : undefined}
+        >
+            {title}
+        </Title2>
+    </Stack>
+);
+
+Default.storyName = 'Title';
+Default.args = {
+    title: 'This is a title',
```

Comment:
I'd set the default arg value for title to empty string and inside each example:
```
{title || 'Title1'}
{title || 'Title2'}
```
This way in the story you know what component are you seeing